main.py

```diff
@@ -28,7 +28,7 @@ def parse_args(args):
     # Modeling.
     parser.add_argument('--device', default='cpu', help='Select cuda for the gpu.')
     parser.add_argument('--model_name', default='lstm_attn')
-    parser.add_argument('--loss_function', default='xent')
+    parser.add_argument('--loss_function', default='xent', choices=['xent','vmf'])
```

Comment:
Nice touch, we should do this for the other options (like mode). Not in this pr though :)

Reply:
Haha, I did that in the other PR (I was playing around there anyway).